kernel/base/src/main/java/com/twosigma/beakerx/kernel/commands/MagicCommand.java

```diff
@@ -190,6 +204,36 @@ private MagicCommandFunctionality classpathAddJar() {
     };
   }
 
+  private void validateWildcardPath(String path) {
+    if (!path.matches("^([^*]*\\*){1}[^*]*$") || !path.endsWith("*")) {
```

Comment:
we are not supporting general globs, just a * at the end, like java classpaths normally do.